apps/functions-subscription/src/main.ts

```diff
@@ -1,21 +1,59 @@
+import { pipe } from 'fp-ts/lib/function';
+import * as E from 'fp-ts/lib/Either';
 import { app } from '@azure/functions';
+import { EventHubProducerClient } from '@azure/event-hubs';
+import { CosmosClient } from '@azure/cosmos';
 import { makeInfoHandler } from './adapters/azure/functions/info';
 import { makePostSubscriptionHandler } from './adapters/azure/functions/subscriptions';
-import { SystemEnv } from './system-env';
+import { makeSubscriptionCosmosContainer } from './adapters/azure/cosmosdb/subscription';
+import { makeSubscriptionRequestEventHubProducer } from './adapters/azure/eventhubs/subscription-request';
+import { parseConfig } from './config';
+import { Capabilities } from './domain/capabilities';
+import { makeSystemEnv } from './system-env';
+import { clock } from './adapters/date/clock';
+import { hashFn } from './adapters/crypto/hash';
 
-/**
- * FIXME: At the moment, we do not have implementations for the env.
- * To let the code properly work, we are creating an empty object.
- */
-const env = {} as unknown as SystemEnv;
+const config = pipe(
+  parseConfig(process.env),
+  E.getOrElseW((error) => {
+    // eslint-disable-next-line functional/no-throw-statements
+    throw new Error(error);
+  }),
+);
+
+const cosmosDB = new CosmosClient(
+  config.subscription.cosmosdb.connectionString,
+);
+
+const subscriptionRequestEventHub = new EventHubProducerClient(
+  config.subscriptionRequest.eventhub.connectionString,
+  config.subscriptionRequest.eventhub.name,
+);
```

Comment:
**note**: Maybe we should use the [Identity Platfrom](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-javascript-get-started?tabs=azure-cli%2Cwindows#connect-using-the-microsoft-identity-platform)